packages/x-license/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.0.0-alpha.2](https://github.com/dtassone/material-ui-x/compare/v0.1.67...v4.0.0-alpha.2) (2020-09-17)
```

Comment:
Noise?

Reply:
Also, I wonder, would it be better to have a single changelog at the root developers can browse? Instead of having to read a dozen of changelogs when they want to update (in the longer term, once we have more packages?

Reply:
Yes I was trying lerna version